src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Content/Controls/FabSamplePage.xaml

```diff
@@ -0,0 +1,94 @@
+﻿<Page x:Class="Uno.Material.Samples.Content.Controls.FabSamplePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:Uno.Material.Samples.Shared.Content"
+	  xmlns:toolkit="using:Uno.UI.Toolkit"
+	  xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:not_win="http://uno.ui/not_win"
+	  xmlns:android="http://uno.ui/android"
+	  xmlns:not_android="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  mc:Ignorable="d not_win android not_android">
+
+	<StackPanel BorderBrush="{StaticResource OnSurfaceBrush}"
+				BorderThickness="1"
+				Margin="12">
+		<TextBlock Text="FLOATING ACTION BUTTONS"
+				   FontSize="20"
+				   FontWeight="Bold"
+				   Foreground="{StaticResource PrimaryBrush}"
+				   Margin="12" />
+
+		<win:Button Style="{StaticResource FabMaterialButtonStyle}"
+					Margin="12">
+			<Path Data="M19 13 13 13 13 19 11 19 11 13 5 13 5 11 11 11 11 5 13 5 13 11 19 11z"
+				  Fill="Black"
+				  HorizontalAlignment="Center"
+				  VerticalAlignment="Center"
+				  Margin="0,0,6,4" />
+		</win:Button>
+
+		<!-- Known Issue, ElevatedView adds white border around fab in WASM -->
+		<not_win:Grid>
+			<toolkit:ElevatedView Elevation="6"
```

Comment:
I would prefer the elevatedView inside the Button's Style (in the library). Does it make sense?

Reply:
I can't apply it there... using toolkit in the library still causes a compile error

Reply:
@NicolasChampagne  PR #46 will solve this!

Reply:
thanks